test/ping-pong/user-local/docker-entrypoint.sh

```diff
@@ -1,6 +1,16 @@
 #! /bin/bash
 set -o errexit
 set -o pipefail
+DEPLOYER_HOST=nightfall_3_deployer_1
 # wait until there's a optimist instance up
 while ! nc -z ${CLIENT_HOST:-client} ${CLIENT_PORT:-80}; do sleep 3; done
+while true; do 
+  if ping -c 1 -W 1 "${DEPLOYER_HOST:-deployer}" > /dev/null 2>&1; then
```

Comment:
Ping pong test did not need to check for deployment because user-local was only started after deployer is done manually. In this case, because user and deployer are started with the same command, I can see why you are checking if deployer is done before starting user-local.

But I am not sure I understand the check.
`ping -c 1 -W 1 "${DEPLOYER_HOST:-deployer}" > /dev/null 2>&1` would not print error messsages to STDOUT or STDERR. But not sure what the if condition is checking here. If the output of ping is correctly redirected to /dev/null